po/de_DE.po

```diff
@@ -128,78 +188,247 @@ msgid "Run '%s --help' to see a full list of available command line options"
 msgstr "'%s --help' ausführen, um die vollständige Parameterliste anzuzeigen"
 
 #. Translators: translate this to give yourself credits.
-#: src/gcolor3-application.c:60
+#: src/gcolor3-application.c:88
 msgid "translator-credits"
 msgstr "Jens John, Vincent Bermel"
 
-#: src/gcolor3-application.c:61
+#: src/gcolor3-application.c:89
 msgid "Website"
 msgstr "Webseite"
 
-#: src/gcolor3-color-store.c:68
+#: src/gcolor3-color-row.c:76
+msgid "Could not parse color to display thumbnail\n"
+msgstr "Farbe konnte nicht verarbeitet werden\n"
+
+#: src/gcolor3-color-selection.c:406
+msgid ""
+"Select the color you want from the outer ring. Select the darkness or "
+"lightness of that color using the inner triangle."
+msgstr ""
+"Wählen Sie die gewünschte Farbe vom äußeren Farbkreis und die gewünschte "
+"Helligkeit auf dem inneren Dreieck."
+
+#: src/gcolor3-color-selection.c:436
+msgid "Picking a color is currently not supported on Wayland."
+msgstr ""
+"Das Abgreifen einer Farbe vom Bildschirm ist momentan nicht unter Wayland "
+"nicht möglich."
+
+#: src/gcolor3-color-selection.c:447
+msgid ""
+"Click the eyedropper, then click a color anywhere on your screen to select "
+"that color."
+msgstr ""
+"Klicken Sie auf das Pipettensymbol und dann auf einen beliebigen Punkt auf "
+"dem Bildschirm um die dortige Farbe auszuwählen."
+
+#: src/gcolor3-color-selection.c:458
+#, fuzzy
+msgid "_Hue:"
+msgstr "Farbwert (_H)"
+
+#: src/gcolor3-color-selection.c:459
+msgid "Position on the color wheel."
+msgstr "Position auf dem Farbkreis."
+
+#: src/gcolor3-color-selection.c:461
+#, fuzzy
+msgid "S_aturation:"
+msgstr "_Sättigung (S)"
+
+#: src/gcolor3-color-selection.c:462
+msgid "Intensity of the color."
+msgstr "Intensität der Farbe."
+
+#: src/gcolor3-color-selection.c:463
+#, fuzzy
+msgid "_Value:"
+msgstr "Dunkelstufe (_V)"
```

Comment:
Is having `(_V)` and such normal in German translations? Note that you may also use `_Dunkelstufe`, just make sure there are no clashes between the mnemonics.

Reply:
i dont think that its normal, looks just like a reference or something

Reply:
Yeah, this is just so I can map the shortcuts as well as the translations to H, S, V which aren't letters found in the translated words. Will need to see if it makes sense that way.

Reply:
If the term HSV is more useful, can't you just not translate them? Otherwise, please do move the mnemonic.

Reply:
Thing is I'm doing the translation blindly. Until next week I don't have access to a machine where I could easily build from master.